src/Components/Moormanage/Boatyards/BoatyardDetailsTable.tsx

```diff
@@ -0,0 +1,68 @@
+import { Button } from "primereact/button";
+import { Column } from "primereact/column";
+import { DataTable } from "primereact/datatable";
+import React from "react";
+import { RowExpansionProps } from "../../../Type/ComponentBasedType";
+
+const BoatyardDetailsTable: React.FC<RowExpansionProps> = ({
+  mooringDetails,
+}) => {
+  if (!mooringDetails || mooringDetails.length === 0) {
+    return null;
+  }
+
+  return (
+    <div style={{ display: "flex", alignItems: "center" }}>
+      <img
+        src="/assets/images/map.png"
+        alt="sea image"
+        style={{ width: "50vw", height: "200px", marginRight:"20px" , marginTop:"-120px" }}
+      />
+      <div className="float-right bg-[#D9D9D9] rounded-md border-[1px] p-1 border-gray-300 w-[22vw]">
+        <DataTable
```

Comment:
change this with custom data table component

Reply:
Not in use, so we are removing this component

Reply:
why it was present originally ?

Reply:
actaully it was creaed as per figma design , figma was changing again and again